airflow/cli/commands/task_command.py

```diff
@@ -320,6 +320,12 @@ def task_test(args, dag=None):
     if not already_has_stream_handler:
         logging.getLogger('airflow.task').propagate = True
 
+    env_vars = {'AIRFLOW_TEST_MODE'}
```

Comment:
Why do we need `AIRFLOW_TEST_MODE` ?

Reply:
This is more to share `AIRFLOW_TEST_MODE` across all executions and provide a standarized env var name.